### ISSUE TITLE
Apache24 and mod_wsgi on Windows Server 2012 R2

### ISSUE THREAD

**Issue by richardewell1862** [GrahamDumpleton/mod_wsgi#462]
Hi Graham,

I am a newbie to Apache, mod_wsgi and Flask so my learning curve has been steep.  I am now stuck though at the final hurdle.

My server is a Windows 2012 R2 machine (an EC2 web server) with Apache24 installed.  I have a simple Flask API I want to deploy.

What I want is get traffic going to app.example.com/friend/friends/v1 to route through to 3.8.16.17/friend/friends/v1, run my Python Flask app and respond with a JSON response.

My Python Flask scripts all work perfectly on localhost.
My service provider’s DNS has been set up to point app.example.com at 3.8.16.17.
On my web server, Apache is set to listen on port 80.
My virtual hosts file (httpd-vhosts.conf) on my web server looks as follows:

   <VirtualHost *:80>
    ServerAdmin enquiries@example.com
    ServerName  app.example.com
    WSGIApplicationGroup %{GLOBAL

    WSGIScriptAlias /friend/friends/v1 "C:/Projects/friend/friends/v1/web.wsgi"
    DocumentRoot C:/Projects/friend/friends /v1
    <Directory "C:/Projects/friend/friends/v1">
        Options -Includes -Indexes -ExecCGI -FollowSymLinks
        AllowOverride None
        Require all granted
    </Directory>
    ErrorLog "C:/Projects/logs/error.log"
    CustomLog "C:/Projects/logs/access.log" common
    </VirtualHost>

If I test this on the web server by putting "3.8.32.18 app.apitude.co.uk" into my hosts file, I can enter app.example.com/friend/friends/v1 in my browser and everything works great.

However, on a different machine, when I try app.example.com/friend/friends/v1 in my browser, I get a "Forbidden You don't have permission to access this resource" error.  When I try 3.8.16.17/friend/friends/v1 in a browser (3.8.16.17 is my web server IP address), everything works great.

What am I missing? What setting am I leaving out?

As an Apache newbie, I need some help.

**Comment by GrahamDumpleton** [comment]
Why are you using '3.8.32.18' in one case and '3.8.16.17' in the other?

**Comment by richardewell1862** [comment]
Sorry, I meant 3.8.16.17 for all.

**Comment by GrahamDumpleton** [comment]
So if was the same IP, you were absolutely certain that local hosts file or DNS on the second machine did map the host name used in the URL to the same IP as work when plugged directly into the browser.

If it wasn't Windows I would suggest using curl and doing:

```
curl -k  http://3.8.16.17 -HHost:app.example.com
```

as alternative to:

```
curl -k  http://app.example.com
```

and see if get same result.

**Comment by richardewell1862** [comment]
Yup, app.example.com/friend/friends/v1 works on my web server (due to the hosts file entry) but does not work from elsewhere.  3.8.16.17/friend/friends/v1 works elsewhere just fine.

OK, I'll try this in curl.
